cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/CassandraDialect.java

```diff
@@ -62,6 +63,9 @@
 import com.datastax.driver.core.querybuilder.Insert;
 import com.datastax.driver.core.querybuilder.QueryBuilder;
 import com.datastax.driver.core.querybuilder.Select;
+import com.google.common.cache.CacheBuilder;
```

Comment:
Interesting, so far we refrained from using Guava in (non-test) code to avoid conflicts with versions of that super-popular library used by users themselves. But given that the C* driver itself uses Guava, I suppose that argument is moot in this case.

@emmanuelbernard, any thoughts?